nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -186,13 +187,17 @@ private int insertWorkflowInstanceWithCte(WorkflowInstance instance) {
     }
   }
 
+  private int getInstancePriority(WorkflowInstance instance) {
+    return instance.priority != null ? instance.priority.intValue() : 0;
```

Comment:
if the column is not null in database, why we need this null check?

Reply:
The method is used for inserting a new workflow, and if `setPriority` has not been called in the builder, the value is null. I thought this is the best place to set this, because there will (in a later PR) come updating a wf priority, where null value in the instance is useful to determine that no changes to priority should be made.

Reply:
Ok